.env.test

```diff
@@ -20,3 +20,8 @@ BROWSER_TESTS_USER_NAME=
 BROWSER_TESTS_USER_PASSWORD=
 BROWSER_TESTS_JWT_SIGN_SECRET=
 VITE_HELSINKI_PROFILE_URL=
+VITE_MATOMO_URL_BASE="//webanalytics.digiaiiris.com/js/"
+VITE_MATOMO_SITE_ID=813
+VITE_MATOMO_SRC_URL="piwik.min.js"
```

Comment:
There is now something wrong in a production (container) build with this. The file is requested from a local (language-) root instead of matomo's own server.

Reply:
When running a development server, there is a this kind of note:

<img width="468" alt="image" src="https://github.com/user-attachments/assets/caf65d5e-eaa9-4f05-ba37-a68d1fbf0ef0">

Reply:
from the `MatomoTracker.ts`, there is a [code like this](https://github.com/jonkoops/matomo-tracker/blob/main/packages/js/src/MatomoTracker.ts#L94), which implicates that the srcUrl parameter should be left as empty.

```
    scriptElement.type = 'text/javascript'
    scriptElement.async = true
    scriptElement.defer = true
    scriptElement.src = srcUrl || `${normalizedUrlBase}matomo.js`

```

Actually, the the README says the same in https://github.com/jonkoops/matomo-tracker/tree/main/packages/react#usage. 
```
const instance = createInstance({
  urlBase: 'https://LINK.TO.DOMAIN',
  siteId: 3,
  userId: 'UID76903202', // optional, default value: `undefined`.
  trackerUrl: 'https://LINK.TO.DOMAIN/tracking.php', // optional, default value: `${urlBase}matomo.php`
  srcUrl: 'https://LINK.TO.DOMAIN/tracking.js', // optional, default value: `${urlBase}matomo.js`
  disabled: false, // optional, false by default. Makes all tracking calls no-ops if set to true.
  heartBeat: { // optional, enabled by default
    active: true, // optional, default value: true
    seconds: 10 // optional, default value: `15
  },
  linkTracking: false, // optional, default value: true
  configurations: { // optional, default value: {}
    // any valid matomo configuration, all below are optional
    disableCookies: true,
    setSecureCookie: true,
    setRequestMethod: 'POST'
  }
})

In this example, the srcUrl is a full absolute URL, not just the path.
```